docs: part 4 - apply style/terminology to top 50 pages (integrate, changelog) (#5302)
n8n-io/n8n-docs: docs/integrations/builtin/core-nodes/n8n-nodes-base.httprequest/README.md

## Templates and examples <a href="#templates-and-examples" id="templates-and-examples"></a>

[Browse HTTP Request integration templates](https://n8n.io/integrations/http-request) or [search all templates](https://n8n.io/workflows/)

## FAQ

> ### Call an API without writing code

Author: ```suggestion
### How do I call an API without writing code?
```

Use the HTTP Request node to query data from any app or service with a REST API. Set the [node parameters](#node-parameters) (method, URL, query parameters, headers, and body), or [import a curl command](#import-curl-command) from the service's API documentation and n8n fills the fields for you.

> ### Authenticate an API request

Author: ```suggestion
### How do I authenticate an API request?
```

Set this up in the [Authentication](#authentication) section. Use a **Predefined Credential Type** when n8n supports the service, since it's the easiest to set up. For other APIs, use generic credentials, which support Basic auth, Header auth, OAuth1, OAuth2, and more. Refer to [HTTP Request credentials](../../credentials/httprequest.md).

> ### Handle an API that returns results across multiple pages

Author: ```suggestion
### How do I handle an API that returns results across many pages?
```
